build.sh

```diff
@@ -0,0 +1,22 @@
+ECOV=0483b1e
+CWD=`pwd`
+
+if [ ! -d eco ]; then
+    git clone https://github.com/softdevteam/eco
+    cd eco
+    git checkout ${ECOV}
+    cd ..
+fi
+
+if [ -f results.csv ]; then
+    rm results.csv
+fi
+
+if [ -f performance.pdf ]; then
+    rm performance.pdf
+fi
+
+cp benchmark.py eco/lib/eco/
+cd eco/lib/eco/
+find $CWD/javastdlib5/ -iname \*.java -exec ./benchmark.py {} \;
```

Comment:
Can't this just be `find . -name "*.java" -exec .benchmark.py {}`? I'm not sure why `CWD` is needed, or what the `\;` bit is doing.

Reply:
Since we are inside the `eco/lib/eco` path, we need `CWD` to find the `javastdlib5` folder. The `\;` is needed for the `exec` argument in `find`. God knows why.

Reply:
Ah, then why not do `find ../../../javastdlib5/`? I must admit to being largely allergic to `pwd`, `cwd` and the like (from bitter experience).

Reply:
I initially did that but `plot.py` which runs in the top-level needs to look at these files again to get their linenumbers in which case `../../` wouldn't work anymore. :(

Reply:
OK, then let's at least not call the variable `CWD` (or `CDW` ;)). Maybe `experimentdir` or something that's a bit more informative and doesn't bring me out in hives?

Reply:
As you wish. ;-)